api-report/driver-definitions.api.md

```diff
@@ -214,7 +214,7 @@ export interface IDriverHeader {
 }
 
 // @public (undocumented)
-export interface IFluidResolvedUrl extends IResolvedUrlBase {
+export interface IFluidResolvedUrl {
```

Comment:
Should this also get deprecated? "Replace usage of `IFluidResolvedUrl` with `IResolvedUrl`"?  Or vice versa -- either way seems we should recommend one or the other and then push to have only one.

Reply:
yeah. it should happen eventually, but its a bigger change, as both are actually used.

Reply:
i also don't know which we want to go with, probably just IResolvedUrl

Reply:
Agree IResolvedUrl - recommend you state that in this change's BREAKING.md to remove ambiguity.  Maybe also followup with a change in `main` to immediately deprecate the other one.

Reply:
i don't want to deprecate is main, as they are not equivalent in main, as i missed the cut off. i will deprecate in next in a follow up, and remove our usages, then is next next we can remove

Reply:
That makes sense, thanks!